### PR TITLE
lang: Update Chinese translations

### PR DESCRIPTION
**Pull request by thttg**
- Remove mistranslations & machine translations in zhCN & zhTW
- Update Chinese translations to the latest version
- Remove Thai from language list

### CODE REVIEW

**Comment by saghul** [comment]
Please drop the changes unrelated to the Chinese translation, we can't merge the PR like that and some of them require discussion.

We will not be renaming English to English (US) for example.

**Comment by thttg** [comment]
> Please drop the changes unrelated to the Chinese translation, we can't merge the PR like that and some of them require discussion.
> 
> We will not be renaming English to English (US) for example.

I restored these changes and only keep "Canadian" to "Canada".

**Review by saghul — APPROVED** [review]

**Comment by thttg** [comment]
The conflicts have been resolved.

**Comment by thttg** [comment]
@saghul Would you mind reviewing it again? The previous one was dismissed because I had to resolve the conflicts.

**Comment by thttg** [comment]
Hi @damencho would you mind reviewing this PR?

**Comment by damencho** [comment]
Jenkins test this please.

**Review by damencho — CHANGES_REQUESTED** [review]

File: lang/languages.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "fa": "Persian",
     "fi": "Finnish",
     "fr": "French",
-    "frCA": "French (Canadian)",
+    "frCA": "French (Canada)",
```
[the remark this answers is not in the document]
Comment (thttg) [reply lang/languages.json]: I dropped it.

**Review by damencho — APPROVED** [review]

**Pull request merged**